src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,26 @@
+.watch {
+  @include square ($watch-size);
+  border: 1px solid black;
+  background-color: #ccc;
```

Comment:
Consider using a variable for the background color to maintain consistency throughout the project.